node/cli/src/chain_spec.rs

```diff
@@ -220,7 +246,8 @@ pub fn testnet_genesis(
 		}),
 		session: Some(SessionConfig {
 			keys: initial_authorities.iter().map(|x| {
-				(x.0.clone(), session_keys(x.2.clone(), x.3.clone(), x.4.clone()))
+				// TODO: Again, is this safe to do?
```

Comment:
What is safe?

Reply:
This comment is related to the comment above asking whether it is safe to use the same seed for `BabeId`, `ImOnlineId` and `AuthorityDiscoveryId`. I guess given that this is only ever used as a  testnetwork it does not matter @bkchr?